### PR TITLE
N25: What's the word

### PR DESCRIPTION
**Pull request by Vrixyz**
_Part of #751_

### CODE REVIEW

**Comment by AngelOnFira** [comment]
I'm wondering if we can add in a gif of it running. Something along these lines:

https://user-images.githubusercontent.com/14791619/132101588-84e9953f-7870-43e3-ab25-ebcb9e2bc44b.mp4

File: content/news/025/index.md [hunk 1]
```diff
@@ -67,6 +67,12 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [What's the word](https://vrixyz.itch.io/whats-the-word)
+
+A UI-based game in which you tap (or miss) buttons.
+
+It's at the prototype stage, let it demonstrate how *small* you can scope your project for a game jam !
```
Comment (AngelOnFira) [inline content/news/025/index.md:74]:
```suggestion
A UI-based game in which you tap (or miss) buttons. It was created with <something>. It's at the prototype stage, but can demonstrate how *small* you can scope your project for a game jam!
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

File: content/news/025/index.md (hunk 1 above)
Comment (AngelOnFira) [reply content/news/025/index.md:74]: It would be cool to hear a bit about what tools were used to create it :)

**Comment by Vrixyz** [comment]
I'd love to add such a gif! I recorded that during development: https://media.discordapp.net/attachments/879816235133575178/879847235578376233/moving_illusion.mov

Your gif has less height so it might be more fitting though !

I'm currently without computer so I might have difficulties adding that gif and resolve conflicts... So feel free to make final edits ; I can try to make them but I'm sorry I can't promise I'll be able to.

**Pull request merged**